elyra/pipeline/handlers.py

```diff
@@ -93,3 +95,26 @@ async def post(self, *args, **kwargs):
         self.set_header("Content-Type", 'application/json')
         self.finish(json_msg)
         self.flush()
+
+
+class PipelineConfigHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to retrieve pipelines editor configuration"""
+
+    valid_resources = ["palette", "properties"]
+
+    @web.authenticated
+    def get(self, resource):
+        if(resource in self.valid_resources):
+            msg_json = self._read_config(resource)
+        else:
+            # invalid resource, throw an error
+            raise ValueError("Invalid configuration name {}".format(resource))
+        self.write(msg_json)
```

Comment:
I think we want to call `self.finish(msg_json)` here.  Should probably set the content-type header as well.

Reply:
`finish` already sets the content-type, but explicitly doing that to follow the same pattern used in other elyra handlers